cli-plugins/manager/manager_unix.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
+
+package manager
+
+var defaultSystemPluginDirs = []string{
+	"/usr/local/lib/docker/cli-plugins", "/usr/local/libexec/docker/cli-plugins",
```

Comment:
I wrote those on the same line because they are "cousins" (`lib` vs `libexec` for different Linux distros, the design has a footnote about that dichotomy).

This is the `defaultSystemPluginDirs`, there is also `defaultUserPluginDir` which is in the user's home dir (next to `config.json`, so obeying `--config` and in `.docker` by default etc). The user variant is not different on Windows so it isn't in this file.